nvflare/app_common/widgets/streaming.py

```diff
@@ -102,11 +102,12 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
             self.engine = fl_ctx.get_engine()
 
     def add(self, tag: str, value, data_type: AnalyticsDataType, global_step: Optional[int] = None, **kwargs):
-        kwargs = kwargs if kwargs else {}
-        global_step = kwargs.get("global_step", None)
-        if global_step:
+        kwargs = kwargs["kwargs"] if kwargs and kwargs["kwargs"] else {}
```

Comment:
I think should be something like: `add(tag, value, data_type, global_step, step=10, hello=100)`
Not `add(tag, value, data_type, global_step, kwargs={"step":10, "hello": 100})`?

what do you think? which call approach you prefer?

Reply:
I just changed this to make the test work, but it was because I didn't add the test case the right way. I am changing this back now.

Reply:
Got it, thanks, LGTM now.